titanic.ipynb

```diff
@@ -5,14 +5,16 @@
       "cell_type": "markdown",
```

Comment:
_Commented on notebook `titanic.ipynb` Cell 2 From line 10 to 13_
```python
# Create a countplot of the man/woman/child classification
sns.countplot(data=titanic, x='who', hue='alive', palette='bright')

# Add some labels and title
```
lines 10 through 13 on right

_Respond and view the context [here](https://gitnotebooks.com/smith-kyle/notebook-diff-tests/pull/25)._